helpers/scraper.js

```diff
@@ -0,0 +1,366 @@
+import request from 'request';
+import cheerio from 'cheerio';
+
+let urls = [];
+// ex. https://www.tfrrs.org/teams/xc/NC_college_m_Duke.html
+let teamBaseUrl = "https://www.tfrrs.org/teams/xc/";
+
+//ex. https://www.tfrrs.org/results/xc/11563.html
+let resultBaseUrl = "https://www.tfrrs.org/results/xc/";
+
+const notTitle = function(s) {
+  if (
+    s == 'Primary Conference' ||
+    s == 'Making Transition' ||
+    s == 'Full Membership' ||
+    s == 'Future Conference'
+  ) {
+    return false;
+  }
+  if (
+    s == 'Savannah State University' ||
+    s == 'California Baptist University' ||
+    s == 'North Alabama !University of North Alabama'
+  ) {
+    return false;
+  }
+  return true;
+};
+
+const getAbbrev = function(state) {
+  let states = [
+        ['Arizona', 'AZ'],
+        ['Alabama', 'AL'],
+        ['Alaska', 'AK'],
+        ['Arizona', 'AZ'],
+        ['Arkansas', 'AR'],
+        ['California', 'CA'],
+        ['Colorado', 'CO'],
+        ['Connecticut', 'CT'],
+        ['Delaware', 'DE'],
+        ['Florida', 'FL'],
+        ['Georgia', 'GA'],
+        ['Hawaii', 'HI'],
+        ['Idaho', 'ID'],
+        ['Illinois', 'IL'],
+        ['Indiana', 'IN'],
+        ['Iowa', 'IA'],
+        ['Kansas', 'KS'],
+        ['Kentucky', 'KY'],
+        ['Kentucky', 'KY'],
+        ['Louisiana', 'LA'],
+        ['Maine', 'ME'],
+        ['Maryland', 'MD'],
+        ['Massachusetts', 'MA'],
+        ['Michigan', 'MI'],
+        ['Minnesota', 'MN'],
+        ['Mississippi', 'MS'],
+        ['Missouri', 'MO'],
+        ['Montana', 'MT'],
+        ['Nebraska', 'NE'],
+        ['Nevada', 'NV'],
+        ['New Hampshire', 'NH'],
+        ['New Jersey', 'NJ'],
+        ['New Mexico', 'NM'],
+        ['New York', 'NY'],
+        ['North Carolina', 'NC'],
+        ['North Dakota', 'ND'],
+        ['Ohio', 'OH'],
+        ['Oklahoma', 'OK'],
+        ['Oregon', 'OR'],
+        ['Pennsylvania', 'PA'],
+        ['Rhode Island', 'RI'],
+        ['South Carolina', 'SC'],
+        ['South Dakota', 'SD'],
+        ['Tennessee', 'TN'],
+        ['Texas', 'TX'],
+        ['Utah', 'UT'],
+        ['Vermont', 'VT'],
+        ['Virginia', 'VA'],
+        ['Washington', 'WA'], 
+        ['District of Columbia', 'DC'],
+        ['West Virginia', 'WV'],
+        ['Wisconsin', 'WI'],
+        ['Wyoming', 'WY'],
+    ];
+
+    for (var i = 0; i < states.length; i++) {
+      if (state == states[i][0]) {
+        return states[i][1];
+      }
+    }
+    return "";
+}
+
+const getCalifornia = function(school) {
```

Comment:
Should be an es6 anonymous function. Not necessary but we should stay consistent with the rest of the codebase.
```javascript
const getCalifornia = (school) => {
...
}
```